wandb/sdk/internal/job_builder.py

```diff
@@ -233,10 +233,8 @@ def _build_artifact_job_source(
                 full_program_relpath = os.path.basename(program_relpath)
         else:
             full_program_relpath = program_relpath
-        entrypoint = [
-            os.path.basename(sys.executable),
-            full_program_relpath,
-        ]
+        python = f"python{'.'.join(runtime.split('.')[:2])}"
```

Comment:
never ever use the current `os` python

Reply:
When we use our default python base images yes.

When users provide a base image, its less clear.

Reply:
lets just add an update from metadata step like repo